atlas/theme.json

```diff
@@ -115,7 +152,7 @@
 	},
 	"styles": {
 		"blocks": {
-			"core/navigation-link": {
```

Comment:
I just removed this but the diff is weird because GH is trying to be too clever.